04.class/Memo.js

```diff
@@ -0,0 +1,16 @@
+export default class Memo {
+  constructor(id, text) {
+    this.id = id;
+    this.text = text;
+  }
+
+  title() {
+    return this.text.match(/^[^\n]+/m)[0];
+  }
+
+  //enquireを使ってメモタイトルを表示させるために、
+  //convert()で{ name: 'memoTitle', value: 'memoID' }に変換する
```

Comment:
コメントの `//` の後ろには空白を入れましょう。

```suggestion
  //  enquireを使ってメモタイトルを表示させるために、
  //  convert()で{ name: 'memoTitle', value: 'memoID' }に変換する
```

Reply:
[7ed801f](https://github.com/Judeeeee/js-practices/pull/4/commits/7ed801ff5f907c0aed3d52b8f68097b470dcf199)で修正しました。